raster.py

```diff
@@ -26,6 +26,11 @@ class AmazonGeoTiff:
   image_mask_array: np.ndarray # ndarray of uint8
   masked_image: np.ma.masked_array
   yearly_masked_image: np.ma.masked_array
+  name: str = ""
```

Comment:
What is this for?

Reply:
this is the name of the raster, which becomes the name of the column when a feature is added